index.html

```diff
@@ -198,9 +198,9 @@ <h2>Get started</h2>
         dolore magna.</article>
       <form action="https://formspree.io/f/mjvdryev" method="POST">
         <ul class="the-relative">
-         <li class="tname"><input type="text" maxlength="30" placeholder="Your name" required></li> 
-         <li class="mail"><input type="email" placeholder="Your email" required></li> 
-         <li class="texts"><input type="text" maxlength="500" class="text-area" placeholder="Enter your message here.." required></li> 
+         <li class="tname"> <h3>Your name</h3><input type="text" maxlength="30" placeholder="Your name" required></li> 
+         <li class="mail"><h3>Your email</h3><input type="email" placeholder="Your email" required></li> 
+         <li class="texts"><h3>Message</h3><input type="text" maxlength="500" class="text-area" placeholder="Enter your message here.." required></li> 
         </ul>
```

Comment:
- [ ] With your great skill of attention to details, you are almost there :muscle: :vulcan_salute: 
- There are still a few things to fix on your form as requested by the previous reviewer:
  - [ ] Did you notice how the `border-color` of the form inputs are `grey` (you can get the color value on Figma), and only changes to `deep black` after receiving `focus`? Kindly implement this.
  - [ ] Another thing to observe is the `padding` or space between the edges on the form input elements and user's input. Spacing ensures readability and avoids misconception. It even makes the design more user-friendly.
  - [ ] Use appropriate `font` properties` and `alignment` for your button text.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/222930614-07a0be97-76db-4e64-81f5-c5447fcfb9ae.png)|![image](https://user-images.githubusercontent.com/99715640/222930622-e0f86dbc-12f3-4f8f-8f63-97f200e7459e.png)|